### PR TITLE
[w43] FredLin: 10/19 ~ 10/23

### PR DESCRIPTION
**Pull request by gasolin**
change Q4 Goals. `low storage` is descoped from 2.5 so will change to measure any feature that during this time frame

### CODE REVIEW

File: FredLin.md [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
+  - r+, Bug 1214533 - Move History and Cokie button down in Browsing Privacy panel
+  - r+, Bug 1193910 - check for updates : nothing happens
+  - r+, Bug 1216400 - Convert settings call sub-panels to use gaia-switch
+  - r+, Bug 1202974 - [Settings] Convert call settings switches to use web components
+    - learn & tested call related functions like FDN, call baring, call forwarding
+  - r+, Bug 1014442 - [Settings] refactor Find My Device panel with AMD pattern
+  - r+, Bug 1020699 - Follow-up to 908300: remove auto-resizing from font_size_utils.js once apps use Header Web Component
 
 ## 2015 Q4 Goals
-1. Solving Blockers and implement low storage features, as measured by implementation completeness.
```
Comment (timdream) [inline FredLin.md]: Could you ask around about the new timeline on this feature? We could still work on it post 2.5 and ship it on 2.6.

Comment (gasolin) [inline FredLin.md]: Yes, scott and I are still tracking on it, but currently people are working on other priority tasks and need time to get some response

Comment (timdream) [inline FredLin.md]: Please update this to

> 1. Solving Blockers and implement <to be decided> features, as measured by implementation completeness.

Comment (gasolin) [inline FredLin.md]: done

Comment (timdream) [inline FredLin.md]: You'll need to escape the tag with `&lt;` and `&gt;` ....

Comment (gasolin) [inline FredLin.md]: Ooh I don't know that can't be shown well on markdown. changed.

**Comment by timdream** [comment]
Thanks.

**Pull request merged**